tests/docker/tox.ini

```diff
@@ -32,8 +32,9 @@ commands =
     docker-compose rm -v -f
```

Comment:
I think we don't need this step or should be executed instead of two steps above.

Reply:
We might need for MySQL and postgres as we are creating volumes in that but I'll check.